content/influxdb/cloud-dedicated/write-data/line protocol/_index.md

```diff
@@ -122,13 +122,13 @@ After setting up InfluxDB and your project, you should have the following:
 The following example shows how to construct `Point` objects that follow the [example `home` schema](#example-home-schema), and then write the points as line protocol to an
 InfluxDB Cloud Dedicated database.
 
-{{< code-tabs-wrapper >}}
-{{% code-tabs %}}
+{{< tabs-wrapper >}}
```

Comment:
Why the change?

Reply:
`code-tabs` are meant to be attached to code blocks rather than tabulating paragraph content.